src/ImageSharp/Formats/Jpeg/Components/Decoder/ColorConverters/JpegColorConverter.FromCmykVector8.cs

```diff
@@ -40,14 +37,14 @@ protected override void ConvertCoreVectorizedInplace(in ComponentValues values)
                     ref Vector<float> y = ref Unsafe.Add(ref yBase, i);
                     Vector<float> k = Unsafe.Add(ref kBase, i) * scale;
 
-                    c = (c * k) * scale;
-                    m = (m * k) * scale;
-                    y = (y * k) * scale;
+                    c = c * k * scale;
+                    m = m * k * scale;
+                    y = y * k * scale;
```

Comment:
```suggestion
                    k = k * scale;
                    c = c * k;
                    m = m * k;
                    y = y * k;
```
The JIT [doesn't CSE this](https://sharplab.io/#v2:EYLgxg9gTgpgtADwGwBYA0AXEBDAzgWwB8ABAJgEYBYAKGIAYACY8gOgDkBXfGKASzFwBuGvSasAShwB2GXtxYBhCPgAOvADY8AyjwBu/GEJoiAzE1IMFDAN40G9pmYBm6iNgwMAstgQA1bOocMDYMAOYwGIIMKny67sG4EVEAvnYOafbEZsQoXuQAFLBODL4wYBjQADwubhgAfAxgAEJ4MGgMRSVlFVDVru4N+C2J7Z2l5VU1AwwAnsNtHTDF4z19tQ0A1vPtUrwyDFIAlAwZNqcODHFQDLhgAcEAvAcwAO5dE71T9fnkDAD0DAwAAteLgWN4/AEgodhNQLhdzg4nNAGPldvteAwnnQopjKgdcQBqQmHRH2Wxw+FUsbdSb9eqNLGLYoAVSkuGwThgLAAggATPmFJaNbYMXgwsnwmkfNbTfBMzpsjlc3kCoXFIatdri2FU6nClZ09azBXCpWc7n8wWdOZasUSyl6hyGz70zZM80qq3qhhbO3ihgAKhud00sMlFzATKjwY2QZD911Toc8qe8tj8duiYjDhmTLzGeDWbDktSjoYZYR5ayTFynlIPpdsoZzTt0tWX0GovbRumtpGzPeHbdvtF6I8RxO5YpTquCc0TKkryHve+vwBwNB4J8/kCMAdTslyOuaL2Hkx2NxDHxUiJJMlM+T9h7ruNUaeivZFtV1uFrYHOo5s+Bq0q+cqmqyX5emqnSagBB5PoOTadiaH5mlBlowcK/YLIB5ZOshI5xk8nqYb+xR+vBmahjASZPkBvpMnGRY0XRybvoysZsU6qYMOmvrcXqeZPAWAmlqcZbJEAA=) (?!). So some `mulps` instructions can be saved.